### PR TITLE
Remove lerna and yarn workspaces

### PR DESCRIPTION
**Pull request by farukaydin**

### CODE REVIEW

**Comment by kuba618** [comment]
lerna is also mentioned in repository-structure.md in docs

File: package.json [hunk 1]
```diff
@@ -4,28 +4,10 @@
   "private": true,
   "scripts": {
     "start": "lerna run --stream --parallel --scope=@*/{web,backend} dev",
```
[the remark this answers is not in the document]
Comment (farukaydin) [reply package.json]: @kuba618 I just removed this file completely, since we will not need it anymore.

**Comment by farukaydin** [comment]
> lerna is also mentioned in repository-structure.md in docs and in .gitignore

@kuba618 adjusted those places.

**Comment by kuba618** [comment]
Documentation needs to be updated now.
As "yarn install" in the root directory will do nothing.

**Comment by kuba618** [comment]
.devcontainer boot.sh also needs to be updated if will be used

**Review by barinali — APPROVED** [review]

**Pull request merged**